pom.xml

```diff
@@ -186,6 +187,28 @@
             </activation>
             <build>
                 <plugins>
+                    <plugin>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>resources</id>
+                                <phase>generate-resources</phase>
+                                <goals>
+                                    <goal>unpack</goal>
+                                </goals>
+                                <configuration>
+                                    <artifactItems>
+                                        <artifactItem>
+                                            <groupId>nl.knaw.dans.easy</groupId>
+                                            <artifactId>easy-licenses</artifactId>
+                                            <version>${easy-licenses.version}</version>
```

Comment:
A local build logs
```
[INFO] --- maven-dependency-plugin:3.0.2:unpack (resources) @ easy-deposit-agreement-creator ---
[INFO] Configured Artifact: nl.knaw.dans.easy:easy-licenses:1.0.1:jar
[INFO] Unpacking /Users/jokep/.m2/repository/nl/knaw/dans/easy/easy-licenses/1.0.1/easy-licenses-1.0.1.jar to /Users/jokep/git/service/easy/easy-deposit-agreement-creator/target/easy-licenses with includes "" and excludes ""
[INFO] 
```
but travis doesn't and subsequently breaks test. What's wrong?

Reply:
i have just released 1.0.2 of easy-licenses
try that first?